File: samples/python/src/Primitives.py

# UI settings
# Expression: x
#   Type: int
#     Layout: None
#     Save: Object Attributes

# Expression: x1
#   Type: float
#     Layout: None
#     Save: Object Attributes

class Primitives:
    @staticmethod
    def run_main():
        Primitives().start()

    def start(self):
        x: int = 1
        x = 2
        x = 3

        x1: float = 1.1
        x1 = 2.2
        x1 = 3.3

        print("done")


Primitives().run_main()

File: samples/csharp/src/Primitives.cs
using System;

// UI settings
// Expression: x
//   Type: int
//     Layout: None
//     Save: Object Attributes

// Expression: x1
//   Type: float
//     Layout: None
//     Save: Object Attributes
class Primitives
{
    public static void RunMain(string[] args)
    {
        new Primitives().Start();
    }

    public void Start()
    {
        int x = 1;
        x = 2;
        x = 3;

        float x1 = 1.1f;
        x1 = 2.2f;
        x1 = 3.3f;

        Console.WriteLine(x + "," + x1);
        Console.WriteLine("done");
    }
}